src/_help/hubs/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
+
+With hubs, your API ecosystem is displayed on a single page, where documentation can be sorted by categories.
+
+Hubs also provide a unified changelog, making it easy to identify all changes across all the documentation it contains at a glance. Search is also global, allowing you to search across all documentation.
+
+Next step: [Create your first hub](/hubs/create-and-manage-hubs).
```

Comment:
This link is broken, and I'm not sure we should add a `.` at the end

Reply:
👍